public_html/wp-content/mu-plugins/jetpack-tweaks/css-sanitization.php

```diff
@@ -244,12 +244,19 @@ function get_custom_css_properties_safelist() {
 		array(
 			'aspect-ratio',
 			'background-blend-mode',
+			'box-orient',
 			'isolation',
+			'line-clamp',
```

Comment:
It looks like this [only works with the prefixed version](https://caniuse.com/css-line-clamp), which would still be stripped out. Should `-webkit-line-clamp` be added too?

Reply:
Ah the magic of devtools, missed that! I've pushed a commit with the `-webkit-*` prefix for the three properties that'll need 'em.